runtime/wasm/src/host_exports.rs

```diff
@@ -549,6 +552,53 @@ where
         BigDecimal::from_str(&s)
             .map_err(|e| HostExportError(format!("failed to parse BigDecimal: {}", e)))
     }
+
+    pub(crate) fn data_source_create(
+        &self,
+        ctx: &mut EventHandlerContext,
+        name: String,
+        params: Vec<String>,
+    ) -> Result<bool, HostExportError<impl ExportError>> {
```

Comment:
In what case does this return `Ok(false)`?

Reply:
Never really, let's make this return `void`. Any error here is fatal.

Reply:
Done